bundle/cluster-logging-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+---
+packageName: clusterlogging
```

Comment:
needs a dash:  ``` operators.operatorframework.io.bundle.package.v1: cluster-logging```

Reply:
updated in https://github.com/openshift/cluster-logging-operator/pull/3142/commits/888d1689be2c50d6e7146e5541d3a11d2a16b894